From 9a95028eac3bb01a4c04012a62a4588d9eed5558 Mon Sep 17 00:00:00 2001
From: Yifan Hong <elsk@google.com>
Date: Mon, 18 Jun 2018 21:22:49 +0000
Subject: [PATCH 04/29] Revert "healthd: add stem to
 health@2.0-service.override [DO NOT MERGE]"

This should have been done with actual fixes to sepolicy and rc scripts.

This reverts commit 299276785787a35effa829f3f9d507cbde266dea.

Reason for revert: broke the build

Bug: 110228707

Change-Id: I23944d044dd2f87e6c1033ed7a4c7cf34f16247e
---
 healthd/Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/healthd/Android.bp b/healthd/Android.bp
index 0f6e8372c..cefe09d5e 100644
--- a/healthd/Android.bp
+++ b/healthd/Android.bp
@@ -61,7 +61,6 @@ cc_binary {
 
 cc_binary {
     name: "android.hardware.health@2.0-service.override",
-    stem: "android.hardware.health@2.0-service",
     defaults: ["android.hardware.health@2.0-service_defaults"],
 
     overrides: [
-- 
2.17.1

